src/main/java/org/threadly/litesockets/UDPServer.java

```diff
@@ -145,5 +164,20 @@ public void run() {
     }
     
   }
+  
+  /**
+   * 
+   * @author lwahlmeier
+   *
+   */
+  private static class WriteDataPair {
```

Comment:
If you wanted you could change this to threadly's tuple class:
http://threadly.github.io/threadly/javadocs/4.5.0/org/threadly/util/Pair.html

`Pair<ByteBuffer, InetSocketAddress>`